lightwood/mixers/nn.py

```diff
@@ -193,11 +191,11 @@ def _fit(self, train_ds, test_ds, stop_training_after_seconds=None):
                         self.criterion_arr.append(torch.nn.BCEWithLogitsLoss(weight=weights_slice))
                         self.unreduced_criterion_arr.append(torch.nn.BCEWithLogitsLoss(weight=weights_slice, reduce=False))
                 elif output_type == COLUMN_DATA_TYPES.NUMERIC:
-                    self.criterion_arr.append(QuantileLoss(quantiles=self.quantiles))
-                    self.unreduced_criterion_arr.append(QuantileLoss(quantiles=self.quantiles, reduce=False))
-                else:
                     self.criterion_arr.append(torch.nn.MSELoss())
                     self.unreduced_criterion_arr.append(torch.nn.MSELoss(reduce=False))
+                else:
+                    self.criterion_arr.append(torch.nn.torch.nn.MSELoss())
+                    self.unreduced_criterion_arr.append(torch.nn.torch.nn.MSELoss(reduce=False))
```

Comment:
Shouldn't this be a single `else:` clause? Also, typo: `torch.nn.torch.nn`

Reply:
I prefer to keep it explicit but... should be a comment about why no and else